### DISCUSSION TITLE
VS Code SQLPage Extension

### DISCUSSION THREAD

**Discussion by YunusMechery** [sqlpage/SQLPage#553] [Ideas]
Hi,
Is there a official SQLPage VSCode extension for autocompletion of SQL, SQLPage components, top level parameters and row level parameters?
Thanks,
Yunus

**Comment by lovasoa** [comment]
hello! There is no such thing at the moment. Would you be interested to work on this?

**Comment by YunusMechery** [comment]
Workaround available here 
https://github.com/sqlpage/SQLPage/discussions/1199

Snippets generation.
